compiler/rustc_smir/src/stable_mir/abi.rs

```diff
@@ -430,6 +430,8 @@ pub enum CallConvention {
     PreserveMost,
     PreserveAll,
 
+    Custom,
```

Comment:
Not directly related to this PR, but why is this enum not marked as `#[non_exhaustive]`?

Reply:
Yeah I was thinking the same. How can this be "stable mir" when we have to make breaking changes like this.